bundles/org.openhab.binding.bondhome/src/main/java/org/openhab/binding/bondhome/internal/handler/BondDeviceHandler.java

```diff
@@ -561,10 +583,14 @@ private synchronized void deleteExtraChannels(List<BondDeviceAction> currentActi
             }
         }
         // Remove power channels if we have a dimmer channel for them;
-        // the dimmer channel already covers the power case
+        // the dimmer channel already covers the power case.
+        // Add the raw channel for advanced users unless we're a ceiling fan.
         if (availableChannelIds.contains(CHANNEL_FAN_SPEED)) {
             availableChannelIds.remove(CHANNEL_POWER);
             availableChannelIds.remove(CHANNEL_FAN_POWER);
+            if (devInfo != null && devInfo.type == BondDeviceType.CEILING_FAN) {
+                availableChannelIds.add(CHANNEL_RAW_FAN_SPEED);
```

Comment:
It's added when ceiling fan, but the comment above says "add (...) unless we're a ceiling fan"?

Reply:
I reserved the logic, but forgot to reverse the entire statement around it.  It used to be "unless we're a FP fan."  I'll get that fixed up.